docs/application/vstools/index.md

```diff
@@ -34,10 +34,14 @@ Visual Studio Tools for Tizen provides Tizen-specific tools to improve your prod
 
   The Log Viewer allows you to check the logs of your application based on the logging methods you have inserted into your code. It helps you to debug your application by capturing all the events logged by the platform and your application.
 
-- [Profiler](tools/profiler.md)
+- [Profiler](tools/profiler-user-manual.md)
 
   The Profiler allows you to analyze the performance of your application. You can record, manage, and view profiling sessions.
 
+- [Memory Profiler](tools/memory-profiler-user-manual.md)
+
+  This is a utility which collects memory profiling data and save them to a file, this part of the Memory Profiler runs on the target side. Also it analyzes the resulting data, the command-line one and the GUI-based one, these analyzing tools run on the host side.
```

Comment:
Please review the rephrase and check whether the meaning of the content is not changed:

Memory Profiler is an utility, which collects memory profiling data and saves them to a file. Usually, the Memory Profiler runs on the target side. In addition, it analyzes the resulting data, from both command-line and GUI-based. These analyzing tools runs on the host side.